rfd/0156-notifications.md

```diff
@@ -162,83 +186,87 @@ message ByRoles {
 }
 ```
 
-##### Unique Notification Codes
+##### Unique Notification Types
 
-Each notification will have a code identifying the type of notification it is. This is so that we can handle notifications in the UI
+Each notification will have a type identifying the type of notification it is. This is so that we can handle notifications in the UI
 differently based on their type, such as determining what action is performed when interacting with it and which icon it should have.
 
-###### Example codes
+###### Example types
 
 ```go
-// Notification codes for identifying different notification types.
+// Notification typess for identifying different notification types.
```

Comment:
I don't think you need to express this in Go syntax. The RFD would be a bit easier to read if you just said something like:

```
###### Example types

- `default-informational`: a generic informational notification
- `default-warning`: a generic warning notification
- `access-request-pending`: notification that an access request is awaiting review
- ...
```